abb_irb6640_moveit_config/launch/demo.launch

```diff
@@ -40,8 +38,6 @@
   </include>
 
   <!-- If database loading was enabled, start mongodb as well -->
-  <include file="$(find abb_irb6640_moveit_config)/launch/default_warehouse_db.launch" if="$(arg db)">
-    <arg name="moveit_warehouse_database_path" value="$(arg db_path)"/>
-  </include>
+  <include file="$(find abb_irb6640_moveit_config)/launch/default_warehouse_db.launch" if="$(arg db)"/>
 
 </launch>
```

Comment:
This seems to revert this file back to one where the DB location isn't configurable. This will lead to problems when this package gets released, as the default installation location `/opt/ros/$distro/..` is not world writable. See #60.